packages/edit-site/src/components/navigation-sidebar/index.js

```diff
@@ -18,8 +24,7 @@ const NavigationSidebar = () => {
 		<>
 			<NavigationToggle isOpen={ isNavigationOpen } />
 			<NavigationPanel isOpen={ isNavigationOpen } />
+			<NavigationPanelPreviewSlot />
```

Comment:
Cool, this seems like a better place for the slot, than nested within the panel. 👍